sql-schema/183.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE users_prefs DROP PRIMARY KEY;
+ALTER TABLE users_prefs DROP INDEX pref;
+INSERT INTO `users_prefs` (`user_id`, `pref`, `value`) SELECT `user_id`, 'dashboard', `dashboard` FROM `users` WHERE `dashboard` != '0';
+INSERT INTO `users_prefs` (`user_id`, `pref`, `value`) SELECT `user_id`, 'twofactor', `twofactor` FROM `users` WHERE `twofactor` != '0';
+ALTER TABLE `users` CHANGE `updated_at` `updated_at` TIMESTAMP NOT NULL DEFAULT '1970-01-01 00:00:01';
```

Comment:
Not really sure how best to handle, looks like I've actually done 1970-01-02 00:00:01 in sql-schema/036.sql before